README.md

```diff
@@ -1082,7 +1082,7 @@ Note: you could follow a similar process to disable the models in this package f
 
 models:
   dbt_project_evaluator:
-    +enabled: "{{ env_var('ENABLE_DBT_PROJECT_EVALUATOR', 'true') }}"
+    +enabled: "{{ env_var('ENABLE_DBT_PROJECT_EVALUATOR', 'true') == 'true' | as_bool }}"
```

Comment:
Thanks!

Thinking about it, it might even be slightly more foolproof by adding a `| lower`, so `True`, `TRUE` and `true` are all considered True.

```suggestion
    +enabled: "{{ env_var('ENABLE_DBT_PROJECT_EVALUATOR', 'true') | lower == 'true' | as_bool }}"
```